Sources/Web3Core/RLP/RLP.swift

```diff
@@ -112,14 +112,13 @@ public struct RLP {
         return encoded.bytes[0]
     }
 
-    // FIXME: Make encode generic to avoid casting it's argument to [AnyObject]
```

Comment:
Removed this comment as I do not see a flexible enough solution for us to implement proper generics when we operate on an array that will hold not just one generic type but multiple different types at once. 
This is where `Any` really shines so to speak.

Reply:
I see the type `Any` as the last resort but if we are careful with it everything should go smoothly.